common/rya.api/src/main/java/org/apache/rya/api/RdfCloudTripleStoreConfiguration.java

```diff
@@ -72,15 +72,21 @@
     public static final String CONF_STATEMENT_METADATA_PROPERTIES = "statement.metadata.properites";
     public static final String CONF_USE_STATEMENT_METADATA = "use.statement.metadata";
     public static final String STATS_PUSH_EMPTY_RDFTYPE_DOWN = "conf.stats.rdftype.down";
+    public static final String INFER_INCLUDE_ALL_VALUES_FROM = "infer.include.allvaluesfrom";
+    public static final String INFER_INCLUDE_DOMAIN_RANGE = "infer.include.domainrange";
+    public static final String INFER_INCLUDE_HAS_SELF = "infer.include.hasself";
+    public static final String INFER_INCLUDE_HAS_VALUE = "infer.include.hasvalue";
     public static final String INFER_INCLUDE_INTERSECTION_OF = "infer.include.intersectionof";
     public static final String INFER_INCLUDE_INVERSEOF = "infer.include.inverseof";
     public static final String INFER_INCLUDE_ONE_OF = "infer.include.oneof";
+    public static final String INFER_INCLUDE_PROPERTY_CHAIN = "infer.include.propertychain";
+    public static final String INFER_INCLUDE_REFLEXIVEPROP = "infer.include.reflexiveprop";
```

Comment:
Nit. Change ...REFLEXIVEPROP to ...REFLEXIVE_PROP to match the style of the rest of the variable names.